### PR TITLE
[Modules] Updated EventHub Namespaces to new dependencies approach & extended readme script 

### PR DESCRIPTION
**Pull request by MrMCake**
# Description

- Updated EventHub Namespaces to new dependencies approach
- Extended readme script to work with empty bicep parameter blocks

| Pipeline |
| - |
| [EventHub: Namespaces](https://github.com/Azure/ResourceModules/actions/workflows/ms.eventhub.namespaces.yml) |

# Type of Change

Please delete options that are not relevant.

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Update to documentation

### CODE REVIEW

File: utilities/tools/Set-ModuleReadMe.ps1
Comment (eriqua) [inline utilities/tools/Set-ModuleReadMe.ps1:652]: Kindly remove changes not in the scope of this PR

Comment (MrMCake) [reply utilities/tools/Set-ModuleReadMe.ps1:652]: Actually it was. It is in the title and description 😉

Comment (MrMCake) [reply utilities/tools/Set-ModuleReadMe.ps1:652]: Will merge set change back from main once that PR is merged

Comment (MrMCake) [reply utilities/tools/Set-ModuleReadMe.ps1:652]: I will merge back from main once that PR ist merged

Comment (MrMCake) [reply utilities/tools/Set-ModuleReadMe.ps1:652]: I will merge back from main once that PR ist merged

Comment (MrMCake) [reply utilities/tools/Set-ModuleReadMe.ps1:652]: I wish there'd be a native feature to set one PR as a predecessor to another. As this one would be blocked without the fix that comes with it. Meh... Maybe one day

Comment (MrMCake) [reply utilities/tools/Set-ModuleReadMe.ps1]: done

File: modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep:153]:
```suggestion
        privateDnsZoneGroup: {
```

File: modules/Microsoft.EventHub/namespaces/.test/pe/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/pe/deploy.test.bicep:46]:
```suggestion
        privateDnsZoneGroup: {
```

File: modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep:6]:
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes.')
```

File: modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep:10]:
```suggestion
@description('Optional. The location to deploy resources to.')
```

File: modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/common/deploy.test.bicep:13]:
```suggestion
@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
```

File: modules/Microsoft.EventHub/namespaces/.test/min/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/min/deploy.test.bicep:6]:
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes.')
```

File: modules/Microsoft.EventHub/namespaces/.test/min/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/min/deploy.test.bicep:10]:
```suggestion
@description('Optional. The location to deploy resources to.')
```

File: modules/Microsoft.EventHub/namespaces/.test/min/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/min/deploy.test.bicep:13]:
```suggestion
@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
```

File: modules/Microsoft.EventHub/namespaces/.test/pe/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/pe/deploy.test.bicep:6]:
```suggestion
@description('Optional. The name of the resource group to deploy for testing purposes.')
```

File: modules/Microsoft.EventHub/namespaces/.test/pe/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/pe/deploy.test.bicep:10]:
```suggestion
@description('Optional. The location to deploy resources to.')
```

File: modules/Microsoft.EventHub/namespaces/.test/pe/deploy.test.bicep
Comment (eriqua) [inline modules/Microsoft.EventHub/namespaces/.test/pe/deploy.test.bicep:13]:
```suggestion
@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
```

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**
